week2/materials/homework/assignments.md

```diff
@@ -4,51 +4,61 @@
 
 Sync your changes to s3 and write down your s3 bucket url.
 
-bucket url: `?`
+bucket url: `http://keerthi-week2.s3-website-us-east-1.amazonaws.com`
 
 **Assignment 5:**
 
 Right now, the website does not support https. Explain which other AWS service needs to be integrated in order to achieve this.
 
-service name: `?`
+service name: `use CloudFront to serve HTTPS requests for my Amazon S3 bucket`
 
 **Assignment 6:**
 
 Figure out and write down the price per month of storing 51TB on S3.
 
-Price: `?`
+<!-- S3 Standard storage: 51 TB per month x 1024 GB in a TB = 52224 GB per month
+Pricing calculations
+Tiered price for: 52224 GB
+51200 GB x 0.0230000000 USD = 1177.60 USD
+1024 GB x 0.0220000000 USD = 22.53 USD
+Total tier cost: 1177.60 USD + 22.53 USD = 1200.1280 USD (S3 Standard storage cost)
+S3 Standard cost (monthly): 1,200.13 USD -->
+
+Price: `1,200.13 USD`
```

Comment:
I Will accept that answer, but it can also be calculated as 1TB = 1000GB if you use base 10 and not base 2 as you do here :) 👌